### PR TITLE
BZ2068283 - Adding clarification for IPsec

### PR DESCRIPTION
**Pull request by kelbrown20**
For Versons 4.8+
[Bugzilla](https://bugzilla.redhat.com/show_bug.cgi?id=2068283)

Preview:

- [Installing a cluster on vSphere with user-provisioned infrastructure](https://deploy-preview-44577--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/installing-vsphere.html#installation-network-user-infra_installing-vsphere)
- [Installing a cluster on any platform](https://deploy-preview-44577--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_platform_agnostic/installing-platform-agnostic.html#installation-network-user-infra_installing-platform-agnostic)
- [IPsec encryption configuration](https://deploy-preview-44577--osdocs.netlify.app/openshift-enterprise/latest/networking/ovn_kubernetes_network_provider/about-ipsec-ovn.html#network-connectivity-requirements-when-ipsec-is-enabled)

### CODE REVIEW

**Comment by lalan7** [comment]
Missing the:  IP proto 50 ESP

**Comment by rbbratta** [comment]
@kelbrown20  

```
.Ports used for all-machine to all-machine communications
```
```
|Protocol
|Port
|Description
```
```
|ESP
|N/A
|IPsec Encapsulating Security Payload (ESP)
```

**Comment by rbbratta** [comment]
I suppose we could do

```
|ESP (50)
|N/A
|IPsec Encapsulating Security Payload (ESP)

```

But persons using IPsec should already know that ESP is IP proto 50.

**Comment by rbbratta** [comment]
```
.3+|UDP
|`500`
|IPsec IKE packets

|`4500`
|IPsec NAT-T packets
```

File: modules/installation-network-user-infra.adoc [hunk 1]
```diff
@@ -183,6 +183,11 @@ The {op-system-base} KVM host must be configured to use bridged networking in li
 ====
 endif::ibm-z-kvm[]
 
+[IMPORTANT]
+====
+When IPsec is enabled on your {product-title} cluster, you must add the UDP 500 IPSEc IKE packets with the UDP 4500 IPsec NAT-T packets on all of your nodes.
```
Comment (rbbratta) [inline modules/installation-network-user-infra.adoc:188]: I not entirely sure about this blurb, we might as well add the ports to the regular list below.

Also `IPsec` is always spelled with lowercase `sec`

**Comment by kelbrown20** [comment]
> I'm not entirely sure about this blurb, we might as well add the ports to the regular list below.

@rbbratta I apologies for my confusion, I was unsure how to format the info in this bug at first. I took out the blurb and just have the list, which I think is a more efficient display of the info. I just updated this PR, thank you for the feed back as well!

**Comment by kelbrown20** [comment]
@rbbratta last thing, this change is not exclusive to vSphere and any platform upi, correct?

**Comment by rbbratta** [comment]
@kelbrown20 The rules need to be added on for any platform UPI that OVNKubernetes supports (platform exceptions below)

We don't have an explicit matrix, but IPsec is an OVNKubernetes feature supported on UPI and IPI where-ever OVNKubernetes is supported.

**Exceptions:** Currently IBMcloud platform does not support IPsec. See https://github.com/openshift/openshift-docs/blame/main/modules/nw-operator-cr.adoc#L297 https://issues.redhat.com/browse/SDN-2629   IBMcloud UPI is not supported yet.

**Comment by kelbrown20** [comment]
Gotcha, thank you!

**Comment by kelbrown20** [comment]
@rbbratta Just updated the section mentioned in BZ, PTAL. Thank you!

**Pull request merged**